README.md

```diff
@@ -261,10 +261,12 @@ The following tools can be used to visualize genomic data or for constructing cu
 - **[IGV js](https://github.com/igvteam/igv)** - Java-based browser. Fast, efficient, scalable visualization tool for genomics data and annotations. Handles a large variety of formats. [ [paper-2019](https://pubmed.ncbi.nlm.nih.gov/31099383) | [web](https://software.broadinstitute.org/software/igv) ]
 - **[Island Plot](https://github.com/lairdm/islandplot)** - D3 JavaScript based genome viewer. Constructs SVGs. [ [paper-2015](https://pubmed.ncbi.nlm.nih.gov/25916842/) ]
 - **[JBrowse](https://github.com/GMOD/jbrowse)** - JavaScript genome browser that is highly customizable via plugins and track customizations. [ [paper-2016](https://pubmed.ncbi.nlm.nih.gov/27072794) | [web](http://jbrowse.org/) ]
+- **[MA plot](https://michaelaupetit.shinyapps.io/EnhancedMAplot_V2/)** - R-Shiny MA plot for interactively selecting/filtering/grouping/exporting genes based on MA data. [ [paper-2020](http://biovis.net/2020/files/BioVis2020__Interactive_MA_plot%20-%20Ali%20Sheharyar.pdf) | [web](https://github.com/alisheharyar/Enhanced_MA_Plot) ]
```

Comment:
There are many shiny apps out there. This section is reserved more for specialized software to address visualzation.